src/main/java/rex/task/Task.java

```diff
@@ -2,7 +2,7 @@
 
 public class Task {
     private static int numberOfTasks = 0;
-    private String description;
+    private final String description;

```

Comment:
Should the constant be in screaming snake case instead?

Reply:
Thank you for the suggestion, I believe the description is actually not a constant, as it is first assigned a String when a Task constructor is called! Constants would be in the form static final instead :)